compiler/compiler/src/compiler.rs

```diff
@@ -146,6 +146,10 @@ impl<N: Network> Compiler<N> {
 
         self.do_pass::<SsaForming>(SsaFormingInput { rename_defs: false })?;
 
+        self.do_pass::<WriteTransforming>(())?;
```

Comment:
Were these passes and the extra changes needed to support execution tests outputting bytecode?